dashboard.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
can't get what this file for

Reply:
Also index.js in project

Reply:
dashboard.html - this file opens when the user correctly writes email and password on the signin page (empty file).
index.html - this file was need, maybe, for write routers code.

Reply:
Ok